README.md

```diff
@@ -112,14 +112,14 @@ $ license-scanner --help
 Usage:
   license-scanner [flags]
 
-
 Flags:
   -g, --acceptable          Flag acceptable
-      --addAll string       Add the licenses from SPDX unzipped release
+      --addAll string       Add licenses
```

Comment:
thanks @markstur 👍 

I think we might want to add a little bit more description here similar to what we have in the PR description:

```
--addAll to work with custom or spdx (embedded) or customPath or spdxPath (not embedded)
```

Or what exactly this option does, addAll can be set to a path with the SPDX released templates or custom licenses.